common/src/require.ts

```diff
@@ -0,0 +1,17 @@
+let result: NodeRequireFunction | (() => null);
+
+if (!!module['require']) {
```

Comment:
I don't understand this line. 
if `module['require']` exists, then the platform is not supported, but inside the if block `module['require']` is assigned to the result, so it should exist?